### PR TITLE
Definition model

### PR DESCRIPTION
**Pull request by richardswei**
generated the model, created the definitions for the controller, and created a basic form on the same page as the entry page. Conversion from rails magic to using jQuery posts to follow

### CODE REVIEW

File: app/assets/javascripts/definitions.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (spinnercat) [inline app/assets/javascripts/definitions.coffee]: what is this file O.O

File: app/assets/stylesheets/definitions.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the definitions controller here.
```
Comment (spinnercat) [inline app/assets/stylesheets/definitions.scss]: what is this file O.O

File: app/controllers/definitions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class DefinitionsController < ApplicationController
+	def create 
+		@definition = Definition.new(
+			definition_params
+		)
+		@definition.entry_id = params[:entry_id]
```
Comment (spinnercat) [inline app/controllers/definitions_controller.rb]: where does this param get set? does rails do this for you?

Comment (richardswei) [reply app/controllers/definitions_controller.rb]: its defined on line 11. But it does seem less readable this way. This I'll fix

Comment (richardswei) [reply app/controllers/definitions_controller.rb]: entry_id is set by rails using foreign keys. When I set the belongs_to and has_many relationships, rails gives us the functionality for free

**Pull request merged**